src/components/Grid/Grid.tsx

```diff
@@ -121,6 +121,9 @@ export const Grid: React.FC<GridProps> = ({
             item={item}
             grid={grid}
             onClick={() => onClick(item)}
```

Comment:
We might need a touch listener as well.

```suggestion
            onClick={() => onClick(item)}
            onTouchStart={() => onClick(item)}
```